### ISSUE TITLE
Combining or and simple

### ISSUE THREAD

**Issue by RWOverdijk** [balderdashy/sails-mongo#172]
To keep it simple, these are my criteria:

```javascript
var criteria = {
  or: [
    {
      to: '53c3a4bf738268196a6191a8' 
    },
    { 
      from: '53c3a4bf738268196a6191a8' 
    }
  ],
  thread: '53ce4f94f3d7c5aec34b9761'
};
```

This applies to model `message` which has `to`, `from`, and `thread` associations. This gives me 0 results, where it should give me 1 result. This is what I expect mongo to create (which does return 1 result):

```javascript
db.message.find({ $or: [
  { to: ObjectId('53c3a4bf738268196a6191a8') },
     { from: ObjectId('53c3a4bf738268196a6191a8') } 
  ], 
  thread: ObjectId('53ce4f94f3d7c5aec34b9761')
})
```

**Update:**
I get this when I log `where` and `queryOptions` here: https://github.com/balderdashy/sails-mongo/blob/master/lib/collection.js#L94

```javascript
{ or:
   [ { to: '53c3a4bf738268196a6191a8' },
     { from: '53c3a4bf738268196a6191a8' } ],
  thread: '53ce4f94f3d7c5aec34b9761' }
{ thread: 53ce4f94f3d7c5aec34b9761,
  '$or':
   [ { to: 53c3a4bf738268196a6191a8 },
     { from: 53c3a4bf738268196a6191a8 } ] } { limit: 30, skip: 0 }
{ _id: { _bsontype: 'ObjectID', id: 'SÎOó×Å®ÃKa' } } { limit: 30 }
{ _id: { _bsontype: 'ObjectID', id: 'SÃ¤¿sh\u0019ja¨' } } { limit: 30 }
{ _id: { _bsontype: 'ObjectID', id: 'S½KU+¯\u0015ÕH¨»' } } {}
```

**Comment by sailsbot** [comment]
Thanks for posting, @RWOverdijk.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
